lib/mage/Mage.js

```diff
@@ -410,13 +410,23 @@ Mage.prototype.useModules = function () {
 		this._modulesList.push({ name: name, path: resolved });
 		this._setupQueue.push(name);
 
+		var mod = {};
+
 		try {
-			var mod = require(resolved);
-			this[name] = this.core.modules[name] = mod;
+			mod = require(resolved);
 		} catch (error) {
-			error.module = name;
-			error.moduleFile = resolved;
-			throw error;
+			if (error.code !== 'MODULE_NOT_FOUND') {
+				error.module = name;
+				error.moduleFile = resolved;
+				throw error;
+			}
+
+			this.core.logger.warning.data({
+				moduleName: name,
+				modulePath: resolved
+			}).log('No module found for module');
```

Comment:
That's a bit of a funky way to express this error :)

"No module found for module"